src/containers/Application/index.js

```diff
@@ -6,16 +6,20 @@ import MatchingCard from '../../pages/MatchingPage';
 import Fortune from '../../pages/Fortune';
 import HttpBin from '../../pages/HttpBin';
 import HomePage from '../../pages/HomePage';
+import Quiz from '../../pages/Quiz';
+import Question from '../../pages/Quiz/QuestionWrapper';
 
 import NavBar from '../../components/NavBar';
 
 export default () => (
   <Router>
     <NavBar />
     <Switch>
+      <Route path="/quiz/:testNumber" component={Question} />
```

Comment:
Could you move this route under/above the `/quiz`?
In this case, all of our routes will be in the correct order.

Reply:
Do you mean that I just need to replace lines order?